ExceptionsHomework/src/ro/scit/menu/Menu.java

```diff
@@ -0,0 +1,160 @@
+
+/**
+ * @author Georgiana
+ * Menu Class
+ *
+ */
+
+public class Menu {
+
+    //   SimpleDateFormat sdf = new SimpleDateFormat("ddMMyyyy");
+    Scanner sc = new Scanner(System.in);
+    List<Student> studentList = new ArrayList<>();
+    Student student = new Student();
+    public void display() {
+        int selection = 0;
+        while (selection != 5) {
+            System.out.println(
+                    "Select an option: \n" +
+                            "  1) Add student\n" +
+                            "  2) Delete student\n" +
+                            "  3) retrieve all students with Age ... \n" +
+                            "  4) list students order by Last Name \n" +
+                            "  5) Exit\n "
+            );
+            selection = sc.nextInt();
+            sc.nextLine();
+            switch (selection) {
+                case 1:
+                    System.out.println("Insert the CNP");
+                    String cnp = sc.nextLine();
+                    System.out.println("Insert student's first name: ");
+                    String firstName = sc.nextLine();
+                    System.out.println("Insert student's last name: ");
+                    String lastName = sc.nextLine();
+                    System.out.println("Insert student's date of birth: ");
+                    //String str = sc.nextLine();
+                    try {
+                        System.out.print("Enter month: ");
+                        int month = Integer.parseInt(sc.nextLine());
+                        System.out.print("Enter day: ");
+                        int day = Integer.parseInt(sc.nextLine());
+                        System.out.print("Enter year: ");
+                        int year = Integer.parseInt(sc.nextLine());
+                        DateOfBirth birthDate = new DateOfBirth(month, day, year);
+                        System.out.println("Date of birth is : " + birthDate);
+                        LocalDate currentDate = LocalDate.now();
+                        System.out.println("The age is : " );
+                        int age = DateOfBirth.calculateAge(birthDate, currentDate );
+                        System.out.println(age);
+                        System.out.println("Insert the gender: male or female");
+                        String gender = sc.next();
+                        if (gender.equalsIgnoreCase("male")) {
+                            student.setGender(GenderEnum.MALE.getGender());
+                        } else if (gender.equalsIgnoreCase
+                                ("female")) {
+                            student.setGender(GenderEnum.FEMALE.getGender());
+                        } else {
+                            System.out.println("Invalid option! Please try again.");
+                        }
+                        Student student1 = new Student(cnp,firstName, lastName, birthDate, age, gender);
+                        addStudent(student1);
```

Comment:
And what happens if the input is not valid and the `addStudent()` method throws an `IllegalArgumentException`?
Maybe you should catch that and allow the user to continue with the main menu.